### DISCUSSION TITLE
Primary key of the editors junction collection must be of type UUID!

### DISCUSSION THREAD

**Discussion by AlaaL** [formfcw/directus-extension-flexible-editor#21] [Q&A] [answered]
Hey, nice extension,

I was testing it and did not get it to work, but i am also new to directus so maybe there is somthing i am missing.
When i try to put the flexible editor in a collection and choose an M2A relation i get this error

![image](https://github.com/formfcw/directus-extension-flexible-editor/assets/5410074/7a125799-185c-4b3b-8295-5aad9dc0b438)

**Comment by formfcw** [comment, marked as answer]
Thanks for the question! You will find the answer here: https://github.com/formfcw/directus-extension-flexible-editor#usage-with-relation-nodes
There is also a video that shows the setup!
